modules/binary-authorization/README.md

```diff
@@ -0,0 +1,60 @@
+# Binary Authorization Infrastructure
+
+This module creates the infrastructure and Attestors necessary to generate attestations on image digests.
+
+## Compatibility/Requirements
+
+* GCP Project ID where the project has an active billing account associated with it
+* Terraform version 0.12+
+* Google Kubernetes Engine cluster with "Binary Authorization" enabled
+
+## Usage
+
+```tf
+# Create a Key Ring
+resource "google_kms_key_ring" "keyring" {
+  name     = "my-example-attestor-key-ring"
+  location = var.keyring-region
+  lifecycle {
+    prevent_destroy = false
+  }
+}
+
+# Create Quality Assurance attestor
+module "quality-attestor" {
+  source = "terraform-google-modules/kubernetes-engine/google//modules/binary-authorization"
+
+  attestor-name = "quality-assurance"
+  keyring-id    = google_kms_key_ring.keyring.id
+}
+
+```
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
```

Comment:
hmm weird that these have not populated with the table of inputs/outputs after you ran `make build`. I will check this out.

Reply:
See https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/530#pullrequestreview-417298510